packages/block-library/src/comment-reply-link/index.php

```diff
@@ -56,12 +56,12 @@ function render_block_core_comment_reply_link( $attributes, $content, $block ) {
 		$classes .= 'has-text-align-' . $attributes['textAlign'];
 	}
 
-	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => esc_attr( $classes ) ) );
 
 	return sprintf(
 		'<div %1$s>%2$s</div>',
 		$wrapper_attributes,
-		$comment_reply_link
+		esc_html( $comment_reply_link )
```

Comment:
Not sure this is necessary or the most appropriate function to use...

Reply:
The function `get_comment_reply_link()` is already escaping the url and the attributes [here](https://github.com/WordPress/WordPress/blob/23be3c2b55d6f44b9863d4d8d939627662131dff/wp-includes/comment-template.php#L1752-L1763). 

But there is not an `html` escape. Would be better to add it to the function instead of everywhere? In that case, should we add an `html` escape on all `sprintf` functions?